runbooks/pipeline/sentry-triage.md

```diff
@@ -0,0 +1,27 @@
+# Sentry triage
+> Important Sentry concepts:
+> * [Issue states](https://docs.sentry.io/product/issues/states-triage/)
+> * [Fingerprintinga and grouping](https://docs.sentry.io/product/sentry-basics/grouping-and-fingerprints/)
+> * [Merging issues](https://docs.sentry.io/product/data-management-settings/event-grouping/merging-issues/)
+
+Once a day, we should check Sentry issues created since the prior day, using the following query.
+
+`is:unresolved firstSeen:-24h ` ([saved search link](https://sentry.calitp.org/organizations/sentry/issues/searches/3/?environment=cal-itp-data-infra&project=2&referrer=issue-list&sort=date&statsPeriod=24h))
+
+Categorize those issues and perform relevant steps if the issue is not already assigned.
+
+### A fingerprinting error (i.e. new issue that should’ve been grouped)
+1. Merge the issues together.
```

Comment:
Since merging is not stably undoable right now in Sentry, I think we should have some more explanatory information here about what qualifies for grouping. If it feels hard to elucidate, maybe instead there could be a call to check in with other Data Services team members about whether merging makes sense before performing the merge.

Reply:
ah I was just typing out more in this section; let me push and see what you think